src/containers/course-sections-list/CourseSectionsList.styles.ts

```diff
@@ -20,5 +22,61 @@ export const styles = {
     transform: 'rotate(90deg)',
     color: 'primary.400',
     cursor: 'pointer'
+  },
+  activityButton: {
+    typography: TypographyVariantEnum.Body2,
+    color: 'basic.blueGray',
+    fontWeight: 500,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    m: '0 7px'
+  },
+  activityButtonIcon: {
+    ml: '8px',
+    width: '16px',
+    height: '16px'
+  },
+  activityButtonContainerDefault: {
+    height: '24px',
+    transition: 'height 0.3s cubic-bezier(0, 0, 0.2, 1)',
+    '& .MuiDivider-root': {
+      display: 'none'
+    },
+    '&:hover': {
+      height: '40px',
+      transition: 'height 0.3s cubic-bezier(0, 0, 0.2, 1)',
+      '& .MuiDivider-root': {
+        display: 'flex',
+        ...fadeAnimation,
+        '&:after, &:before': {
+          borderColor: palette.basic.lightGray
+        }
+      }
+    }
+  },
+  activityButtonContainerVisible: {
+    height: '40px',
+    '& .MuiDivider-root': {
+      '&:after, &:before': {
+        borderColor: palette.basic.lightGray
+      }
+    }
+  },
+  menuRoot: {
+    '& .MuiPaper-root': {
+      mt: '12px',
+      '& .MuiMenuItem-root': {
+        p: '12px 16px',
+        typography: TypographyVariantEnum.Body2,
+        color: 'basic.darkGray',
+        fontWeight: 500
+      }
+    }
+  },
+  menuIcon: {
```

Comment:
Has same styles as activityButtonIcon, you can move properties to an object and use spread or use one style property for both cases